Ferry.java

```diff
@@ -3,9 +3,41 @@
  * has a ferry code and information about both the source and
  * the destination island
  *
- * @author (your name)
+ * @author (20067052)
  * @version (a version number or a date)
  */
 public class Ferry {
+    private int ferry_ID;
+    private Island source_island;
+    private Island destination_island;
+
+    public  Ferry(int ferryCode,Island sourceIsland,Island destinationIsland) {
+        ferry_ID=ferryCode;
+        source_island=sourceIsland;
+        destination_island=destinationIsland;
+    }
+    public int getFerryCode(){return ferry_ID;}
```

Comment:
Make sure there's 1 line space between all methods and fields. IntelliJ should do it for you

Reply:
i will use the auto format thingy that u mentioned.